### PR TITLE
Default port for analyzer

### PR DESCRIPTION
**Pull request by smacker**
Part of https://github.com/src-d/lookout/issues/350

### CODE REVIEW

**Comment by smacker** [comment]
The pr needs to be merged & released before updating analyzers (because they use lookout-sdk in CI)

File: cmd/lookout-sdk/event.go [hunk 1]
```diff
@@ -32,9 +32,7 @@ type EventCommand struct {
 	RevFrom    string `long:"from" default:"HEAD^" description:"name of the base revision for event"`
 	RevTo      string `long:"to" default:"HEAD" description:"name of the head revision for event"`
 	ConfigJSON string `long:"config-json" description:"arbitrary JSON configuration for request to an analyzer"`
-	Args       struct {
-		Analyzer string `positional-arg-name:"analyzer" description:"gRPC URL of the analyzer to use"`
-	} `positional-args:"yes" required:"yes"`
+	Analyzer   string `long:"analyzer" default:"ipv4://localhost:9930" description:"gRPC URL of the analyzer to use"`
```
Comment (carlosms) [inline cmd/lookout-sdk/event.go]: With this change the new release will break the usage from/to
- `lookout-sdk review ipv4://localhost:10302`
- `lookout-sdk review --analyzer ipv4://localhost:10302`

right?
Is there a need for this, or could we just make the argument optional to keep compatibility?

Comment (smacker) [reply cmd/lookout-sdk/event.go]: The reason why it's changed is in the commit message: https://github.com/src-d/lookout/pull/389/commits/3275303db234dd6e4e314c2787683a3bdec48d60

I would prefer to use `--analyze` as a recommended way to change address. But return positional argument for compatibility. WDYT?

Comment (carlosms) [reply cmd/lookout-sdk/event.go]: I see your point. Even if it makes the code ugly, I think we should prioritize compatibility and keep the argument.
And if we keep the argument having also `--analyzer` to do the same thing will lead to unnecessary confusion.

I would just go with the optional argument, checking in our code if it's missing to use the default, and adding to the argument description `  (default: ipv4://localhost:9930)`

**Comment by smacker** [comment]
@carlosms updated. I squashed all commits together and pushed force.

**Comment by carlosms** [comment]
The tests are failing, but I don't think it's related to the changes in this PR (please double check in case I'm wrong).
I created an issue to track it, https://github.com/src-d/lookout/issues/391

**Comment by smacker** [comment]
True. This test became unstable (only in travis on linux though, I can't reproduce locally on mac or I have never seen it failing on travis macos). Restart usually helps.

**Comment by smacker** [comment]
Yep. Green now. Merging.

**Pull request merged**